test/Microsoft.DocAsCode.Common.Tests/GitUtilityTest.cs

```diff
@@ -30,6 +33,15 @@ public void Environment_ForBranchName()
         Assert.Equal("special-branch", info.RemoteBranch);
     }
 
+    [Fact]
+    public void Environment_ForGitTimeout()
```

Comment:
We probably don't need to test it as the logic pretty minor. If we do, make the property `internal` instead of using reflection.